parser/tools/generate_ast.cc

```diff
@@ -375,6 +483,60 @@ void emitNodeClassfile(ostream &out, NodeDef &node) {
     out << "    return to_string(buf);\n";
     out << "  }" << '\n';
     out << '\n';
+
+    // toRuby
+    out << "  std::string " << node.name << "::toRuby(const core::GlobalState &gs, int tabs) {" << '\n'
+        << "    fmt::memory_buffer buf;" << '\n';
+    out << "    fmt::format_to(buf, \"s(:" << rubyNodeNameForInternalNodeName(node.name) << "\");" << '\n';
+    // Generate fields
+    for (auto &arg : node.fields) {
+        if (arg.type == Loc) {
```

Comment:
This implicitly requires us to order fields in the same way how whitequark does.
Is it already true today?
I have minor fud that it might be hard to keep them in sync, but I'm happy to wait till this FUD becomes reality.
In the mean time, I guess this is a debugging tool that will make your life better.

Reply:
> This implicitly requires us to order fields in the same way how whitequark does.

Yes, and this order feels right to me (and so I think it makes sense to "backport" it as well). For example, an `If` node has two children: truthy and falsey branches. It sounds correct to declare them as
```
{"then_", Node}, {"else_", Node}
```

> Is it already true today?

Yes, mostly. However there's some difference in the AST format: sorbet has an `assign` node, `int` node has (obviously) a string as the only child, plain string literals without interpolation are always wrapped into `dstr` and maybe more, I didn't have a chance to look into all of them.